maxim/logger/portkey/__init__.py

```
@@ -0,0 +1,3 @@
+from .client import instrument_portkey
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)